src/controllers/comment.controller.js

```diff
@@ -40,10 +45,12 @@ export default class CommentController {
      * @description it responds to user about the update result
      */
     static updateComments = async (req, res) => {
+        const currentUser = req.sessionUser;
         const { commentId } = req.params;
         const { requestId, comment } = req.body;
         const commentData = { id: commentId, requestId, comment };
         const updatedCommentToReturn = await updateComment(commentData);
+        await createInAppNotification(currentUser, requestId, 4);
```

Comment:
Could you use the variables you created in `notificationTypes.utils` instead of `4`?